docs/content/index.md

```diff
@@ -31,6 +31,18 @@ In `config/application.rb`, add **after the application definition**:
 require "primer/view_components/engine"
 ```
 
+To add the JavaScript behaviours, in your `application.html.erb` in the `<head>` tag add:
```

Comment:
```suggestion
Optionally, to add the JavaScript behaviours, in your `application.html.erb` in the `<head>` tag add:
```

Could we call out this is only necessary for certain components? I wonder if this is confusing without more explanation. WDYT?

Reply:
I think this is good :+1: